package.json

```diff
@@ -86,6 +87,7 @@
     "level-js": "^2.2.4",
     "lint-staged": "^4.3.0",
     "lodash": "^4.17.10",
+    "moment": "^2.22.2",
```

Comment:
Does this need to be a dependency in bpanel? bpanel-ui uses it but it lists it as its own dependency and so will package it on its own. I feel like we should try and keep hard deps in the main bpanel repo to things that the architecture needs or that plugins will want to ensure they are all using the same version of (like we do for bcoin).